src/components/CountryItem/CountryItem.test.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { MockedProvider } from '@apollo/client/testing';
+import { render } from '@testing-library/react';
+import { CountryItem } from './CountryItem';
+
+describe('Test CountryItem component', () => {
+  it('should render CountryItem component by class & by props className "test-country-item"', () => {
+    const { container } = render(

```

Comment:
In this case would make sense to check whether the values you pass via the props ("testTitle", "testValue") are rendered correctly into the DOM. You can use getByText to achieve this.